packages/nextra-theme-docs/src/index.tsx

```diff
@@ -178,7 +170,7 @@ const InnerLayout = ({
       : null
 
   const gitTimestampEl = date ? (
-    <div className="pointer-default mt-12 mb-8 block text-right text-xs text-gray-500 dark:text-gray-400">
+    <div className="pointer-default mt-12 mb-8 block ltr:text-right rtl:text-left text-xs text-gray-500 dark:text-gray-400">
```

Comment:
timestamp should be on left side
<img width="900" alt="image" src="https://user-images.githubusercontent.com/7361780/184453315-5891d2e7-82b0-4686-abaf-350173b66d12.png">